App.js

```diff
@@ -50,11 +51,14 @@ import * as Device from 'expo-device';
 import * as ScreenOrientation from 'expo-screen-orientation';
 
 
-YellowBox.ignoreWarnings([
+
+
+LogBox.ignoreLogs([
   'VirtualizedLists should never be nested', // TODO: Remove when fixed
   'Warning: componentWillReceiveProps',
-  'Warning: componentWillMount'
-])
+  'Warning: componentWillMount',
+  'Warning: Each child'
```

Comment:
Should we not ignore this? Because ideally we want to make every child unique by providing a key right? Thought?